ansible/group_vars/all

```diff
@@ -203,6 +203,8 @@ nginx:
 # The key db.whisk.auth is the name of the authentication database where all keys of all users are stored.
 # The db_prefix is defined for each environment on its own. The CouchDb credentials are also defined for each environment on its own.
 db:
+  dir:
+    become: "{{ db_dir_become | default(false) }}"
```

Comment:
is this privilege escalation?

Reply:
Yes if needed.
I just imitated the configuration in nginx.

One more thing to note is, after we deploy couchdb, there would be configuration files.
And if we try to redeploy it, it will fail because couchdb map the configuration files with its own permission(`uid: 1000`). As of now, there is no privilege escalation applied for this.

So, in order to redeploy it, we must remove that directory first.
(If there is no volume mapping for configuration, it's fine.)

Regarding this behavior, I want to discuss which direction is more appropriate.

1. Manually delete all configuration before redeployment.
2. Explicitly delete all configuration before redeployment via Ansible
3. Keep the configuration files and just redeploy couchdb.

One of main purposes of this pr is to make CouchDB restartable.
For some reason, we will change configuration and want to restart it with keeping up existing cluster.

If we change configuration and restart CouchDB manually, there will be no problem.
But if we change and redeploy it via Ansible, we need to consider above situation.

If we decide to redeploy it via Ansible, [serial](http://docs.ansible.com/ansible/latest/playbooks_delegation.html#rolling-update-batch-size) also should be applied as 1, to keep the cluster.